client/src/components/reviews/AddReviewModal/modalHelperFunctions.jsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import clsx from 'clsx';
+import { makeStyles } from '@material-ui/core/styles';
+import { Radio, Typography } from '@material-ui/core';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormLabel from '@material-ui/core/FormLabel';
```

Comment:
could consider condensing this a little more

Reply:
Changed.